x-pack/plugins/apm/public/components/shared/aggregated_transactions_badge/index.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiBadge, EuiToolTip } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function AggregatedTransactionsBadge() {
+  return (
+    <div>
+      <EuiToolTip
+        position="bottom"
+        content={i18n.translate(
+          'xpack.apm.aggregatedTransactions.fallback.tooltip',
+          {
+            defaultMessage: `This page is using transaction event data as no metrics events were found in the current time range, or a filter has been applied based on fields that are not available in metric even documents.`,
```

Comment:
Is it "metrics events" or "metric events"?

Reply:
It was originally `metrics events` but (https://github.com/elastic/kibana/pull/108080#discussion_r686313801) but it might have been an oversight.